scripts/nbconverted/PCA_ICA_model.py

```diff
@@ -0,0 +1,101 @@
+
+# coding: utf-8
+
+# In[1]:
+
+
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+
+# In[2]:
+
+
+#-------------------------------------------------------------------------------------------------------------------------------
+# Input: Pa gene expression data from ArrayExpress (matrix: sample x gene)
+# Data compression method: PCA or ICA
+# Output: Reduced Pa gene expressiond ata (matrix: sample x 2 linear combination of genes)
+#-------------------------------------------------------------------------------------------------------------------------------
+from sklearn.decomposition import PCA, FastICA
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+np.random.seed(123)
+
+
+# In[3]:
+
+
+# load arguments
+data_file = 'C:/Users/alexj/Documents/UPenn/CGreene/Pseudomonas/data/all-pseudomonas-gene-normalized.pcl'
+map_file = 'C:/Users/alexj/Documents/UPenn/CGreene/Pseudomonas/metadata/mapping_sampleID_medium.txt'
```

Comment:
May be use relative path, e.g. data_file =  os.path.join("Pseudomonas", "data", "all-pseudomonas-gene-normalized.pcl")